x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.tsx

```diff
@@ -61,6 +61,9 @@ export const CredentialsList: React.FC = () => {
           </EuiCopy>
         );
       },
+      mobileOptions: {
+        width: '100%',
+      },
```

Comment:
Ideally this produces this (much nicer) mobile UI layout:

<img width="385" alt="_idealworld" src="https://user-images.githubusercontent.com/549407/97337029-f86e1380-183c-11eb-8e23-6f8bce81641b.png">

However - this API isn't in yet. See https://github.com/elastic/eui/issues/4168 as to why the `fullWidth: true` prop doesn't work.

I'll undraft this PR once https://github.com/elastic/eui/pull/4169 is in Kibana.

Reply:
Awesome!